packages/react-instantsearch-core/src/core/indexUtils.js

```diff
@@ -187,14 +187,14 @@ export function getCurrentRefinementValue(
 }
 
 export function cleanUpValue(searchState, context, id) {
-  const indexName = getIndex(context);
+  const index = getIndex(context);
```

Comment:
it seems getIndex is returning an indexId. Shouldn't we make that clear by calling it getIndexId ?

Reply:
Yep I hesitate to update it: the first reason why I didn't do it in this PR it's because it adds more changes to the diff. The other reason is linked to the situation describes above:

> Internally the value exposes though the `Index` context has changed: it's now the `indexId` not the `indexName`. It should not have an impact neither because the places where we use the value from the context we actually want the identifier (we can expose both of them at some point if needed).

I didn't change it to avoid confusion inside the connectors implementations that use the value return by the function. But it seems that it brings more confusion to use `getIndex` rather than `getIndexId`. I'll change it in a dedicated PR.